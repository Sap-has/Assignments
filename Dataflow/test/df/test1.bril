@main {
  a: int = const 5;
  b: int = const 10;
  c: int = add a b;
  d: int = mul a b;
  e: int = add c d;
  print e;
}