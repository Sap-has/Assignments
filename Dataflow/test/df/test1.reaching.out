b1:
  in:  âˆ…
  out: a.b1.0, b.b1.1, c.b1.2, d.b1.3, e.b1.4
